packages/local-environment/src/network.ts

```diff
@@ -39,24 +39,33 @@ export abstract class Network {
   recoveryManager: string;
   weth: string;
 
+  rpc: string[];
```

Comment:
lets make rpc a getter method `get rpcs(): string[]`. I see it as a dynamic param. Fix the following code according to this. In particular no need to have a variable holding it from construction.

Reply:
wait, we already have that. then u can remove the rpc field from network, and make sure that Implementations of Network just query `.rpcs`

Reply:
Removed.